content/docs/pipelines/sdk/python-based-visualizations.md

```diff
@@ -4,174 +4,59 @@ description = "Predefined and arbitrary visualizations of pipeline outputs"
 weight = 80
 +++
 
-This page describes python based visualizations, how to create them, and how to
-use them to visualize results within the Pipelines UI.
+This page describes Python based visualizations, how to create them, and how to
+use them to visualize results within the Kubeflow Pipelines UI.
 
 ## Introduction
 
 Python based visualizations are a new method to visualize results within the
-Pipelines UI. This new method of visualizing results is done through the usage
-of [nbcovert](https://github.com/jupyter/nbconvert), the tool used by Jupyter
-Notebook that generates outputs. Alongside the usage of nbconvert, results of a
-pipeline can now be visualized without a component being included within the
-pipeline itself. The process of visualizing results are now decoupled from a
-pipeline.
+Kubeflow Pipelines UI. This new method of visualizing results is done through
+the usage of [nbcovert](https://github.com/jupyter/nbconvert). Alongside the
+usage of nbconvert, results of a pipeline can now be visualized without a
+component being included within the pipeline itself. The process of visualizing
+results are now decoupled from a pipeline.
 
 Python based visualizations provide two categories of visualizations. The first
-being **predefined visualizations**. These visualizations are curated by the
-Kubeflow Pipelines Team and serve as a way for users to easily and quickly
-generate powerful visualizations. However, these are not limited to just the
-Pipelines Team and this will be discussed further below in
-[Why Predefined Visualizations are Important](/docs/pipelines/sdk/
-python-based-visualizations/#why-predefined-visualizations-are-important).
-The second category is **arbitrary visualizations**. Arbitrary visualizations
-allow for user provided python visualization code to be used to generate
+being **predefined visualizations**. These visualizations are provided by
+default in Kubeflow Pipelines and serve as a way for you and your customers to
+easily and quickly generate powerful visualizations. The second category is
+**arbitrary visualizations**. Arbitrary visualizations allow for you and your
```

Comment:
Use the term **custom visualizations** rather than **arbitrary visualizations**. The word "custom" is more common in this context, and I've just noticed that the UI uses the word "custom" too.